docs/bundles/plain.md

```diff
@@ -29,16 +58,123 @@ manifests
 └── deployment.yaml
 ```
 
+For a bundle `git` repo, any directory that contains only static Kubernetes manifests checked into a `git` repository
+accessible via a remote URL (HTTP or SSH) can be considered a plain bundle and sourced by the plain provisioner. For
+more information on bundles backed by `git` repositories, see the [`git` based bundles doc](../sources/git.md).
+
 > Note: there must be at least one resource in the manifests directory in order for the bundle to be a valid
-> plain+v0 bundle.
+> `plain+v0` bundle.
 
 ## Technical Details
 
-* The static manifests must be located in the /manifests directory for the bundle to be a
-valid `plain+v0` bundle that the provisioner can unpack. A plain bundle image without a /manifests directory is
-invalid and will not be successfully unpacked onto the cluster.
+* The static manifests must be located in the root-level `/manifests` directory in a bundle image for the bundle to be a
+  valid `plain+v0` bundle that the provisioner can unpack. A plain bundle image without a `/manifests` directory is
+  invalid and will not be successfully unpacked onto the cluster.
+* For a bundle `git` repo, the manifests directory can be anywhere in the repository, not just at the root-level. The
+  location can be specified via `spec.source.git.directory`. There must be a flat directory consisting of Kubernetes YAML manifests at the provided location in order to have a valid bundle git repo. If a specific directory is not provided, it is assumed to be `./manifests` in a bundle git repo.
 * The manifests directory should be flat: all manifests should be at the top-level with no subdirectories.
-* Including any content in the /manifests directory of a plain bundle that is not static manifests will result in
-a failure when creating content on-cluster from that bundle. Essentially, any file that would not
-successfully `kubectl apply` will result in an error, but multi-object YAML files, or JSON files, are fine. There will
-be validation tooling provided that can determine whether a given artifact is a valid bundle.
\ No newline at end of file
+* The plain bundle image can be built from any base image, but `scratch` is recommended as it keeps the resulting bundle
+  image a minimal size.
+* It is required that `kubectl apply` is able to process all .yaml files in the directory that make up a plain bundle. For example, multi-object YAML files are acceptable, but Ansible playbooks would not be.
```

Comment:
I would try to rewrite these to be focused on the contents of a plain bundle root directory, and not at all aware of the source implementation. Then the source docs should talk about the specifics of handing a bundle root off to a provisioner.

Reply:
Let me know if https://github.com/operator-framework/rukpak/pull/642/commits/56cfbae769c96755690befe3628c45b9e36ee244 suffices